mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -252,6 +269,10 @@ class ItoFCastOpConversion : public OpConversionPattern<CastOp> {
   matchAndRewrite(CastOp castOp, typename CastOp::Adaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
 
+    Location loc = castOp.getLoc();
```

Comment:
Can you move this closer to its usage?

Reply:
I think the type here is afte type conversion, and there were discussions on EuroLLVM to make emitc have signed instead of signless. So I would interpret both signless and signed here for now (similar as it is done for the arith.addi conversion)